src/sentry/api/serializers/models/project.py

```diff
@@ -66,6 +68,17 @@
 LATEST_DEPLOYS_KEY = "latestDeploys"
 
 
+def _get_team_memberships(team_list: Sequence[Team], user: User) -> Iterable[int]:
```

Comment:
slightly unrelated, but kind of weird we're using `user` here and not `member`.

Reply:
Er, possibly because the `team_list` could span teams on multiple organizations, therefore there wouldn't be only one relevant `OrganizationMember`? (I don't know for sure if that's possible, though.)